requirements/base.txt

```diff
@@ -34,7 +34,8 @@ channels==2.4.0
 daphne==2.5.0
 
 # Plotly
-scipy==1.5.2
+scipy==1.5.2; python_version == '3.7'
+scipy==1.7.3; python_version > '3.7'
 
 # Dash
```

Comment:
This is covered in your other PR, so can be removed from this one. We would just merge your other PR first.